src/traces/funnel/layout_attributes.js

```diff
@@ -0,0 +1,64 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    funnelmode: {
+        valType: 'enumerated',
+        values: ['stack', 'group', 'overlay'],
+        dflt: 'stack',
+        role: 'info',
+        editType: 'calc',
+        description: [
+            'Determines how bars at the same location coordinate',
+            'are displayed on the graph.',
+            'With *stack*, the bars are stacked on top of one another',
+            'With *group*, the bars are plotted next to one another',
+            'centered around the shared location.',
+            'With *overlay*, the bars are plotted over one another,',
+            'you might need to an *opacity* to see multiple bars.'
+        ].join(' ')
+    },
+    funnelnorm: {
```

Comment:
I don't think there's a single mock locking down `funnelnorm` at the moment, could you add one (unless I missed it)?

Reply:
`Funnelnorm` is no longer supported: https://github.com/plotly/plotly.js/pull/3817/commits/2a3c3543ed8fc64016fa16104dfa82715633822a.

Reply:
Nicely done :tada: